joint_trajectory_controller/test/joint_trajectory_controller.test

```diff
@@ -35,5 +35,5 @@
   <test test-name="joint_trajectory_controller_test"
         pkg="joint_trajectory_controller"
         type="joint_trajectory_controller_test"
-        time-limit="85.0"/>
+        time-limit="95.0"/>
```

Comment:
Is this necessary?

Reply:
Yes, the additional test takes some more seconds. I think, some "sleep" in the old tests aren't really needed, I'll try to optimize a bit to stay at 85 seconds here.